src/frameworks/angular/utils.ts

```diff
@@ -420,7 +421,13 @@ export async function getBrowserConfig(sourceDir: string, configuration: string)
   );
   const targetOptions = await architectHost.getOptionsForTarget(buildOrBrowserTarget);
   assertIsString(targetOptions?.outputPath);
-  const outputPath = join(targetOptions.outputPath, buildTarget ? "browser" : "");
+
+  const builderName = await architectHost.getBuilderNameForTarget(buildOrBrowserTarget);
+
+  const outputPath =
+    builderName === ExpectedBuilder.APPLICATION
+      ? join(targetOptions.outputPath, buildTarget ? "browser" : "")
+      : targetOptions.outputPath;
```

Comment:
I think we can keep only one ternary here:
```suggestion
  const outputPath = join(
    targetOptions.outputPath,
    buildTarget && builderName !== ExpectedBuilder.APPLICATION ? "browser" : "",
  );
```

Reply:
oops, right